app/src/main/res/layout/grid_box.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="wrap_content" android:layout_height="match_parent">
+<Button
+    android:id="@+id/button"
+    android:paddingLeft="10dp"
+    android:paddingRight="10dp"
```

Comment:
define all dimensions in dimen.xml, and use generic names for generic dimentions like margins.